agent/address.h

```diff
@@ -60,6 +60,7 @@
 #include <sys/socket.h>
 #include <netinet/in.h>
 #include <arpa/inet.h>
+#include <netdb.h>
```

Comment:
you are right - the problem is **HAVE_NETDB_H** was not included in the config.h for windows which made me not include it to the cmake configure process. I will fix this..